.gitignore

```diff
@@ -1,4 +1,4 @@
-# EMAN2 libs
```

Comment:
Was this uncommented intentionally?

Reply:
No this was an accident, it is back to normal